datacenter/dtr/2.2/guides/admin/upgrade.md

```diff
@@ -18,22 +18,25 @@ support upgrades according to the following rules:
   version. We also strongly recommend upgrading to the latest minor/patch
   version for your major version first.
 
-|From| To| Description| Supported|
-|:----|:---|:------------|----------|
-| 2.2.0 | 2.2.1 | patch upgrade | yes |
-| 2.2.0 | 2.2.2 | skip patch version | yes |
-| 2.2.2 | 2.2.1 | patch downgrade | no |
-| 2.1.0 | 2.2.0 | minor upgrade | yes |
-| 2.1.1 | 2.2.0 | minor upgrade | yes |
-| 2.1.2 | 2.2.2 | minor upgrade | yes |
-| 2.0.1 | 2.2.0 | skip minor version | no |
-| 2.2.0 | 2.1.0 | minor downgrade | no |
-| 1.4.3 | 2.0.0 | major upgrade | yes |
-| 1.4.3 | 2.0.3 | major upgrade | yes |
-| 1.4.3 | 3.0.0 | skip major version | no |
-| 1.4.1 | 2.0.3 | major upgrade from an old version | no |
-| 1.4.3 | 2.1.0 | major upgrade skipping minor version | no |
-| 2.0.0 | 1.4.3 | major downgrade | no |
+| Description | From | To | Supported |
+|:------------|:-----|:---|-----------|
+| patch upgrade | x.y.0 | x.y.1 | yes |
+| skip patch version | x.y.0 | x.y.2 | yes |
+| patch downgrade | x.y.2 | x.y.1 | no |
+| minor upgrade | x.y.* | x.y+1.* | yes |
+| skip minor version | x.y.* | x.y+2.* | no |
+| minor downgrade | x.y.* | x.y-1.* | no |
+| skip major version | x.*.* | x+2.*.* | no |
+| major downgrade | x.*.* | x-1.*.* | no |
+
+In the next table, y and z are the latest minor or patch version  and a,b the
```

Comment:
Since we're not using `a` and `b` you can remove this sentence, and make this a single table again. After that and padding the table, we're ready to ship.

Pro tip: if you're using Atom, there's this `markdown-table-formatter` that automatically fixes tables when you save.